airbyte-cdk/java/airbyte-cdk/dependencies/src/main/resources/log4j2-test.xml

```diff
@@ -13,53 +13,48 @@
 
     <Appenders>
         <Console name="JvmLogsStdOut" target="SYSTEM_OUT">
```

Comment:
cleaning things up by centralizing the ContextMapFilter inside the Routing appender